osquery/tables/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ else()
   ADD_OSQUERY_LINK(FALSE "cryptsetup")
   ADD_OSQUERY_LINK(FALSE "udev")
   ADD_OSQUERY_LINK(FALSE "uuid")
+  ADD_OSQUERY_LINK(FALSE "ip4tc")
```

Comment:
We'll need to build this statically in the future. This dynamic lib does not exist on the system by default.